### DISCUSSION TITLE
recognized digital ROIs correct, raw value wrong

### DISCUSSION THREAD

**Discussion by schneich** [jomjol/AI-on-the-edge-device#3252] [Q&A]
Hi everybody,

I have re-done my casing for my gasmeter. The image is now better, with less reflections on the glas and on the numbers.
Even though I have improved the image quality, I get funny miss-readings:

If I have a look at the recognized digital ROIs, everything seems correct.
![Screenshot_2024-09-24-15-51-48-39_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/7906ab1a-ff3e-422b-90fb-29fa7c6b7670)

But the raw value then is wrong.
![Screenshot_2024-09-24-15-52-15-47_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/f29327e3-1517-4c0a-9abd-c646f90db1e1)

Is there a way to improve my configuration, so that the values are not over-corrected by some algorithm?

Any hints are very welcome,
Chris

p.s.: this is version 15.7.0

**Comment by caco3** [comment]
Check the documentation regarding the ROIS. I think you might need to make them smaller.
